### ISSUE TITLE
Unable to reproduce results (TF variant)

### ISSUE THREAD

**Issue by iamgroot42** [IBM/dstc7-noesis#22]
Hey

I ran this code with the given hyper-parameters, same glove-vector embedding, sand dataset pointed to in the link (Ubuntu corpus). However, the results I have got so far (been training for more than a week) are not even close to the numbers given in the README. Am I missing something? 

Also, for some reason, the validation loss seems to be sharply **increasing** with iterations, which is very odd. Is there something wrong with the way the model is being trained/the code?

How many epochs was this model run for?

**Comment by kpfadnis** [comment]
@iamgroot42 Thanks for the question. We will try to re-run the baseline experiments to verify the reported numbers and shed more light on the validation loss behavior. 

Please note, since we are fast approaching the submission deadline, at this time we cannot promise swift resolution.

**Comment by basma-b** [comment]
@kpfadnis @chulakar I am trying to produce results on **test** set, but I'm unable to get the same results on **dev** as the ones you report. Can you please share the **checkpoint** directory of your model ? so I will just load it and make sure I produce **test** results ?
